site/content/docs/commands/svc-show.en.md

```diff
@@ -10,13 +10,23 @@ $ copilot svc show
 ## What are the flags?
 
 ```
-  -a, --app string        Name of the application.
-  -h, --help              help for show
-      --json              Optional. Output in JSON format.
-      --manifest string   Optional. Name of the environment in which the service was deployed;
-                          output the manifest file used for that deployment.
-  -n, --name string       Name of the service.
-      --resources         Optional. Show the resources in your service.
+-a, --app string        Name of the application.
```

Comment:
Unindenting is fine, but this undoes the changes made in https://github.com/aws/copilot-cli/pull/3784. 🥺 Made "Outputs" -> "Output" for `--json` and `--manifest`, and added wording to `--manifest`

Reply:
Reverted! sorry I didn't even realize, I blindly copy pasted from the cli's help menu :P